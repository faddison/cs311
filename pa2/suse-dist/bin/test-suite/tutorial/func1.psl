defvar f = lambda() { print("not till later\n") } in {
print("now\n");
f();
}
