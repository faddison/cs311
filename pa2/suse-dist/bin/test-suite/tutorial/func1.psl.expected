now
not till later
not till later

